torch/_higher_order_ops/invoke_subgraph.py

```diff
@@ -232,12 +246,14 @@ def _(ctx, subgraph, identifier, operands):
     return ctx.wrap_tensors(out)
 
 
+@invoke_quant_tracer.py_impl(FakeTensorMode)
 @invoke_subgraph.py_impl(FakeTensorMode)
 def _(mode, subgraph, identifier, operands):
     # TODO(anijain2305) - Implement fake tensor caching.
     return subgraph(*operands)
 
 
+@invoke_quant_tracer.py_impl(ProxyTorchDispatchMode)
```

Comment:
This is causing invoke_quant to show up as invoke_subgraph after AOTDispatcher. To match how other HOPs work, it should show up as an invoke_quant